packages/block-library/src/navigation-link/block.json

```diff
@@ -29,6 +29,15 @@
 			"type": "string"
 		}
 	},
+	"usesContext": [
+		"core/textColor",
+		"core/customTextColor",
+		"core/backgroundColor",
+		"core/customBackgroundColor",
+		"core/fontSize",
+		"core/customFontSize",
+		"core/showSubmenuIcon"
+	],
```

Comment:
I notice the query block isn't using the core/ prefix. Probably not needed.

Reply:
Ah yep looks like that as intentional – see https://github.com/WordPress/gutenberg/pull/22199#discussion_r424667982. I'll remove `core/`.

Reply:
83e5f3508ba7f6eaff0c510d83472d0bfb7c3bc8